### ISSUE TITLE
Apps named with accented characters disappear when using the unaccented character to search

### ISSUE THREAD

**Issue by RedBearAK** [Ulauncher/Ulauncher#708]
Ulauncher 5.10.0 (API 2.0.0) 
Linux Mint 20.1, 64-bit
English (US) keyboard layout

I just installed an application called "Touché". I haven't even figured out how to type accented characters on Linux yet, I literally had to copy and paste that from the web. 

If I search in Ulauncher for "touch", it appears. 

If I continue on to type "touche" (without the accented character, just the normal "e" character), "Touché" suddenly disappears from the Ulauncher list. As if there is no match. 

Albert launcher and the Cinnamon menu, by contrast, both correctly continue showing me the "Touché" application when I type out "touche" without the accented character. 

Bottom line: Ulauncher needs to learn to include names with accented characters in search results when the "base" unaccented character is used. This is how it works in every other app launcher or file search application I've had experience with. 

The accented characters are just variations on the unaccented character. So the matching must be done on a version of the name that is stripped of all accents. The user should be able to search for "man" and see an application called "Mañana". And again I don't yet know exactly how to type that character, so I had to past the proper accented version from the web.

**Comment by friday** [comment]
I think this is just due to the broken implementation of the fuzzy matching logic. It's concatenating the name of the app and the name of the binary and then checks for how similar your query is to that whole string, instead of checking both separately. It's fixed in my fork.

**Comment by friday** [comment]
This is fixed in the Ulauncher v6 branch (see the PR linked above). It won't be released for a while, but we made a tracking issue if you want to be notified #869
